report_csv/models/ir_report.py

```diff
@@ -55,7 +55,7 @@ def _render_csv(self, report_ref, docids, data):
         report_sudo = self._get_report(report_ref)
         report_model_name = "report.%s" % report_sudo.report_name
         report_model = self.env[report_model_name]
-        res_id = len(docids) == 1 and docids[0]
+        res_id = docids[0] if docids and len(docids) == 1 else None
```

Comment:
just replaced 
`res_id = (len(docids) == 1 and docids[0]) if docids else False`
with
`res_id = docids[0] if docids and len(docids) == 1 else None`

wich is more readable

Reply:
@petrus-v @remi-filament 
Just added a test and the little change above